### PR TITLE
Tabs in use

### PR DESCRIPTION
**Pull request by ar4s**
Connected to #1492

### CODE REVIEW

File: src/ralph/admin/mixins.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from __future__ import unicode_literals
 
 import reversion
+from collections import namedtuple
```
Comment (mkurek) [inline src/ralph/admin/mixins.py:8]: import order (reversion is 3rd party package)

File: src/ralph/admin/sites.py [hunk 2]
```diff
@@ -68,6 +68,23 @@ def get_urls(self, *args, **kwargs):
                 ))
         return urlpatterns
 
+    def get_extra_view_menu_items(self):
+        from collections import defaultdict
```
Comment (mkurek) [inline src/ralph/admin/sites.py]: why do you import here?

File: src/ralph/admin/templates/data_center/datacenterasset/network.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{% extends BASE_TEMPLATE %}
```
Comment (vi4m) [inline src/ralph/admin/templates/data_center/datacenterasset/network.html:1]: why only datacenterasset ? What about the backoffice devices?

File: src/ralph/admin/sitetrees.py [hunk 4]
```diff
@@ -27,7 +22,42 @@ def ralph_item(*args, **kwargs):
...
+        for view in extra_views[name]
+    ]
+
+
+def section(section_name, app, model):
+    app, model = map(str.lower, [app, model])
+    return ralph_item(
+        title=section_name,
+        url='admin:{}_{}_changelist'.format(app, model),
+        access_by_perms='{}.change_{}'.format(app, model),
```
Comment (ghost) [inline src/ralph/admin/sitetrees.py:41]: Is this the only place where we calculate `url` and `access_by_perms`?

File: src/ralph/admin/sites.py [hunk 5]
```diff
@@ -63,11 +65,32 @@ def get_urls(self, *args, **kwargs):
...
                     name='{}_{}_{}'.format(*get_urls_chunks(model, view))
                 ))
         return urlpatterns
 
+    def get_extra_view_menu_items(self):
+        """Method returns list of items for sitetree mechanism."""
+        items = defaultdict(list)
+
+        def get_item(model, view, change_view=False):
+            url = 'admin:{}_{}_{}'.format(
```
Comment (ghost) [inline src/ralph/admin/sites.py:78]: This is next time I see such code, maybe we could add a helper for that?
Simple function is enough. What do you think?

File: src/ralph/admin/templatetags/ralph_tags.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
```
Comment (szok) [inline src/ralph/admin/templatetags/ralph_tags.py]: Remove '**future**' *

File: src/ralph/data_center/admin.py [hunk 7]
```diff
@@ -4,6 +4,7 @@
 from ralph.admin import RalphAdmin, register
 from import_export.admin import ImportExportModelAdmin
```
Comment (szok) [inline src/ralph/data_center/admin.py:4]: order import :)

File: src/ralph/data_center/admin.py [hunk 8]
```diff
@@ -38,10 +39,23 @@ class DataCenterAdmin(RalphAdmin):
...
+        context = super(NetworkView, self).get_context_data(**kwargs)
+        context['adresses'] = IPAddress.objects.filter(asset=self)
+        return context
+
+
 @register(DataCenterAsset)
 class DataCenterAssetAdmin(ImportExportModelAdmin, RalphAdmin):
     """Data Center Asset admin class."""
 
+    change_views = [NetworkView]
```
Comment (szok) [inline src/ralph/data_center/admin.py]: recommended from doc: user decorator :)

File: src/ralph/admin/sitetrees.py (hunk 4 above)
Comment (ar4s) [inline src/ralph/admin/sitetrees.py:41]: yes, at this moment

**Comment by szok** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**